### ISSUE TITLE
Form labels and/or select form elements are not clearing previous elements as they should

### ISSUE THREAD

**Issue by GoogleCodeExporter** [davidbelow/blueprintcss#42]
```
Labels and select form elements are not clearing as they should.

See in the tests.html sample form the section for dummy 3 and dummy 4.

        <div class="column span-2">
        <label for="dummy3">dummy3 select</label>
        </div>

        <div class="column prepend-1 span-5 last">
        <select id="dummy3" name="dummy3">
            <option value="1">Ottawa</option>
            <option value="2">Calgary</option>
            <option value="3">Moosejaw</option>
        </select>
        </div>

        <div class="column span-2">
        <label for="dummy4">dummy4 select</label>
        </div>

        <div class="column span-6 last">
        <select id="dummy4" name="dummy4">
            <option>Favorite pet</option>
                <optgroup label="mammals">
                    <option>dog</option>
                    <option>cat</option>
                    <option>rabbit</option>
                    <option>horse</option>
                </optgroup>
                <optgroup label="reptiles">
                    <option>iguana</option>
                    <option>snake</option>
                </optgroup>
        </select>
        </div>


```

Original issue reported on code.google.com by `glenn.re...@gmail.com` on 8 Sep 2007 at 9:27

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in the upcoming forms.css file.
```

Original comment by `ola...@gmail.com` on 12 Sep 2007 at 4:48

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.6.
```

Original comment by `ola...@gmail.com` on 29 Sep 2007 at 12:55
- Changed state: **Fixed**
